eth/tracers/api.go

```diff
@@ -937,5 +939,11 @@ func APIs(backend Backend) []rpc.API {
 			Service:   NewAPI(backend),
 			Public:    false,
 		},
+		{
+			Namespace: "eth",
```

Comment:
```suggestion
			Namespace: "debug",
```

Reply:
If use `eth` we can reuse `eth_getBlockResultByHash` api.

Reply:
be careful with debug_namespace. It contains many dangerous RPC APIs which should not be exposed to public

Reply:
It's probably cleanest if we expose such APIs under a `scroll_` namespace, I imagine we might have more custom APIs in the future.

Reply:
Let's move this to a new `scroll_` namespace in a separate PR. (cc @icemelon @HAOYUatHZ).

Reply:
why new PR?

Reply:
I'd rather use `scroll_` since compactness is not a priority here and that's more consistent with existing APIs (`admin_`, `clique_`, etc also don't use abbreviations). But `scr_` works too.

True, if it's not a big change, the namespace can also be updated in this PR.

Reply:
I don't have a strong opinion towards `scr_` or `scroll_`. Both look good to me.

Reply:
Have a second thought. let's just `scroll_` as the namespace.